source/optional_features/prefill_user_details/index.html.md.erb

```diff
@@ -0,0 +1,47 @@
+---
+title: Prefilling fields on the payment page
+weight: 137
+---
+
+# Prefilling fields on the payment page
+
+If you collect a user's details in your service before you redirect them to GOV.UK Pay, you can prefill some of the fields on the __Enter card details__ page. If you prefill fields, users should not need to enter their details twice.
+
+Users can still edit these details before they complete their payment.
+
+To prefill fields, include the following optional parameters in the <a href="https://govukpay-api-browser.cloudapps.digital/#newpayment" target="blank">API request</a> when you [create a new payment](/payment_flow/#making-a-payment):
+
+- `email`
+- `prefilled_cardholder_details` - with optional `cardholder_name` and `billing_address` parameters
+
+For example:
+
+```javascript
+"email": "sherlock.holmes@example.com",
+"prefilled_cardholder_details": {
+    "cardholder_name": "Sherlock Holmes",
+    "billing_address": {
+        "line1": "221 Baker Street",
+        "line2": "Flat b",
+        "postcode": "NW1 6XE",
+        "city": "London",
+        "country": "GB"
+ }
+```
+
+All the parameters in `billing_address` are optional.
+
+If you include the `country` parameter, it must be an <a href="https://www.iso.org/iso-3166-country-codes.html" target="blank">ISO 3166 code</a>. If the parameter is invalid or missing, the __Country or territory__ field on the payment page will default to 'Great Britain'.
+
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more information.
+
+## Data collected by your GOV.UK Pay admin account
+
+After the user completes their payment, your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) will collect the user's:
+
+- email address - even if you've disabled collecting your users' email addresses
+- cardholder name
+- billing address - if you've enabled collecting your users' billing addresses
```

Comment:
should this say "disabled" [billing address]? also @SandorArpa is this still true, since the edge case you found - i.e. front end overwrites address to null/ empty if collecting billing address is disabled in self-service.

Reply:
We store the billing address fields if any/all sent to the API. If the service is set NOT to collect billing address when (and only when) the user clicks continue on the card details page to confirm the payment, the billing address details are overwritten to empty fields in connector by frontend. If the charge stays in created state (eg the payment never been materialised) or it's being cancelled on the card details page then the billing address is not overwritten in connector.